### ISSUE TITLE
Create standalone SoundDescriptors instead of using vanilla forms.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Gambling#122]
A standalone form will allow other plugins to modify the sounds that games use without affect the rest of Fallout 4.

**Comment by TheWhiteCollarPlayers** [comment]
Agreed, a good path to follow, will be necessary for custom sfx as well

**Comment by Scrivener07** [comment]
This is now done on the `implementations` branch.

**Comment by TheWhiteCollarPlayers** [comment]
Have you committed this? Can we close this?

**Comment by Scrivener07** [comment]
No, its on `implementations` branch anyway. Issues dont get closed unless the issue is fixed on the `master` branch.

**Comment by TheWhiteCollarPlayers** [comment]
Ahhh ok I follow, makes sense
